src/semantic-ui/site/elements/segment.overrides

```diff
@@ -66,3 +66,8 @@ Segment Overrides - REACT-INVENIO-APP-ILS
   background: @loaderDimmerColor;
 }
 
+/* backoffice */
```

Comment:
Instead of creating new classes, why not using the [Message](https://react.semantic-ui.com/collections/message/#variations-warning) component of Semantic-UI?